resource-managers/kubernetes/integration-tests/pom.xml

```diff
@@ -34,7 +34,7 @@
     <!-- Please see README.md in this directory for explanation of these -->
     <spark.kubernetes.test.sparkTgz>N/A</spark.kubernetes.test.sparkTgz>
     <spark.kubernetes.test.unpackSparkDir>${project.build.directory}/spark-dist-unpacked</spark.kubernetes.test.unpackSparkDir>
-    <spark.kubernetes.test.imageTag>N/A</spark.kubernetes.test.imageTag>
+    <spark.kubernetes.test.imageTag>kubernetes/dockerfiles/spark/Dockerfile.java17</spark.kubernetes.test.imageTag>
```

Comment:
This PR is still draft due to this. I'll remove this after observing the Jenkins K8s test result.